### ISSUE TITLE
Video exported invalid format!

### ISSUE THREAD

**Issue by phucsang0spt** [etro-js/etro#108]
After output to mp4 video file, i try to open video via firefox or some media player (tried on lap and phone), it show invalid format or only open in some seconds not full video.
But it still open normally on chrome  or VLC player.

 ```
 (movie! as any).startTime = 0;
      const blob = await movie!.record({
        frameRate: 30,
        audio: true,
        video: true,
        // type: mineType,
      });
      const link = document.createElement("a");
      link.href = URL.createObjectURL(blob);
      link.download = orignalName;
      document.body.appendChild(link);
      link.click();
      document.body.removeChild(link);
```

**Comment by phucsang0spt** [comment]
I tried `type: mineType` but seem like it not work, output format always is webm

**Comment by clabe45** [comment]
Should now be fixed in version 0.8.3
